proxy.php

```diff
@@ -272,16 +273,20 @@ function getHttpContentAndStatus($url, $timeout, $user_agent)
 
     // if there's POST data, send our proxy request as a POST
     if (!empty($_POST)) {
-        $postBody = http_build_query($_POST);
+        $postBody = file_get_contents("php://input");
```

Comment:
Looks good to me. @mattab and @diosmosis any thoughts maybe too?

Reply:
Looks good to me too, except possibly for the other comment you made.